### ISSUE TITLE
Swagger UI is broken

### ISSUE THREAD

**Issue by softworkz** [MediaBrowser/swagger-ui#4]
Many of the item schemas are no longer displayed:

![image](https://github.com/MediaBrowser/swagger-ui/assets/4985349/75591135-7c4f-4e4f-96c6-2246232f3553)

Can you just revert your latest update? Was there anything in it that is needed?

**Comment by LukePulverenti** [comment]
What update?

**Comment by softworkz** [comment]
This: https://github.com/MediaBrowser/swagger-ui/commit/e94f5926779d4ea26d40d6727def4cfc8f5806b4

**Comment by LukePulverenti** [comment]
No, it had security fixes. Plus it was three years old. We can't stay on an old build forever. Whatever it is we should just figure it out.
